test_suite/package/test_otel.py

```diff
@@ -0,0 +1,132 @@
+        system_status = cmd_output["InstanceStatuses"][0]["SystemStatus"]["Status"]
+
+        if instance_status == "ok" and system_status == "ok":
+            break
+        logger.info("Instance status is not 'passed' yet. Waiting...")
+        time.sleep(5)
+
+
+@pytest.fixture()
+def modify_iam_role(request, instance_data, host):
+    instance_id = instance_data['instance_id']
+    iam_role_name = "CloudWatchAgentServerRole_2"
+
+    command_to_run = [
+        'aws', 'ec2', 'associate-iam-instance-profile',
+        '--instance-id', instance_id,
+        '--region', 'us-west-1',
+        '--iam-instance-profile', 'Name="{}"'.format(iam_role_name)
+    ]
+    modify_iam_role_cmd = ' '.join(command_to_run)
+
+    assert host.backend.run_local(modify_iam_role_cmd), 'faild to update iam role'
+
+
+@pytest.fixture()
+def install_packages(request, host):
+    class_instance = request.node.cls
+    install_cmd = f'yum install -y  {class_instance.package_name} --nogpgcheck'
+    repo_path = ('https://copr.fedorainfracloud.org/coprs/miyunari'
+                 '/redhat-opentelemetry-collector/repo/rhel-9/miyunari-redhat-opentelemetry-collector-rhel-9.repo')
+    with host.sudo():
+        assert host.run_test(f'yum-config-manager --add-repo {repo_path}')
+        assert host.run_test(install_cmd), f'Failed to install the package {class_instance.package_name}'
+        output = host.check_output("rpm -qa | grep opentelemetry*")
+        print(output)
+
+
+@pytest.fixture()
+def start_service(request, host):
+    class_instance = request.node.cls
+    start_enable_service = (f'systemctl start {class_instance.package_name} && '
+                            f'systemctl enable {class_instance.package_name}')
+    is_active = (f'systemctl is-active {class_instance.package_name}')
+
+    with host.sudo():
+        assert host.run(start_enable_service).succeeded, (f'Failed to start the service {class_instance.package_name}')
+        assert host.run(is_active).succeeded, (f'Failed to activate the service {class_instance.package_name}')
+
+    def finalizer():
+        logger.info(f'Removing the package {class_instance.package_name}')
+        assert host.run(f'sudo yum remove -y {class_instance.package_name}')
+        assert not host.check_output(f'rpm -q {class_instance.package_name}')
+        assert not host.run(f'ssh {class_instance.instance_dns}').succeeded
+        logger.info("Verify logs don't appear")
+        # TODO: verify logs don't appear.
+    request.addfinalizer(finalizer)
+
+
+@pytest.fixture(scope="function", autouse=True)
+def run_subscription_manager_auto(request, host, instance_data):
+    class_instance = request.node.cls
+    # Run the subscription manager auto test before any tests in this file
+    TestsSubscriptionManager.test_subscription_manager_auto(class_instance, host, instance_data)
+
+
+@pytest.mark.package
+@pytest.mark.run_on(['rhel9.1'])
```

Comment:
I guess this will change when we do final testing, to something like `@pytest.mark.run_on(['>=rhel9.5'])`.
Now, maybe we should limit it to rhel9.4+?
```suggestion
@pytest.mark.run_on(['>=rhel9.4'])
```